proposals/lip-0045.md

```diff
@@ -340,28 +391,20 @@ channelSchema = {
                 },
                 "root": {
                     "dataType": "bytes",
+                    "length": HASH_LENGTH,
                     "fieldNumber": 3
                 }
             }
         },
         "partnerChainOutboxRoot": {
             "dataType": "bytes",
+            "length": HASH_LENGTH,
             "fieldNumber": 3
         },
         "messageFeeTokenID": {
```

Comment:
In the description below it says:

>  This property is an object storing a chainID and a localID. These values together form the token ID of the token used to pay for the cross-chain message fees. The default values are chainID = 1 and localID = 0, corresponding to the LSK token.

I think this is outdated and still from the time when token IDs where objects on not fixed byte length identifiers.

Reply:
fixed